OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -17,6 +17,20 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Flags]
+	public enum UncloakType
+	{
+		None = 0,
+		Attack = 1,
+		Move = 2,
+		Unload = 4,
+		Infiltrate = 8,
+		Demolish = 16,
+		Damage = 32,
+		[Desc("Uncloaks on Attack, Unload, Infiltrate, Demolish")]
```

Comment:
I realize you chose this on purpose, but it is an english sentence because this is documentation.
As a reader I don't know if this is AND (imagine that scenario) or OR.

Common sense says it is an or but there's no reason not be a bit more helpful.

Perhaps `Uncloaks on any of ...` then?